### PR TITLE
Update recipe

### PR DESCRIPTION
**Pull request by jdblischak**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

* I already rerendered locally

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,24 +7,22 @@ package:
...
-  fn: reshape_{{ version }}.tar.gz
   url:
     - {{ cran_mirror }}/src/contrib/reshape_{{ version }}.tar.gz
     - {{ cran_mirror }}/src/contrib/Archive/reshape/reshape_{{ version }}.tar.gz
   sha256: 4d5597fde8511e8fe4e4d1fd7adfc7ab37ff41ac68c76a746f7487d7b106d168
 
 build:
-  merge_build_host: true  # [win]
-  number: 2
-  skip: true  # [win32]
```
Comment (jdblischak) [inline recipe/meta.yaml:19]: I don't think we need to explicitly skip win32 builds any more

File: recipe/meta.yaml [hunk 2]
```diff
@@ -7,24 +7,22 @@ package:
...
+  merge_build_host: True  # [win]
+  number: 3
   rpaths:
     - lib/R/lib/
     - lib/
 
 requirements:
   build:
-    - {{ compiler('c') }}        # [not win]
-    - {{ compiler('m2w64_c') }}        # [win]
```
Comment (jdblischak) [inline recipe/meta.yaml:27]: This recipe contains compiled code, but all the builds pass without explicitly specifying the compilers. I assume this means they are being provided by one of the dependencies, presumably r-base

File: recipe/meta.yaml [hunk 3]
```diff
@@ -40,18 +38,29 @@ test:
 about:
   home: http://had.co.nz/reshape
   license: MIT
-  summary: 'Flexibly restructure and aggregate data using  just two functions: melt and cast.'
+  summary: 'Flexibly restructure and aggregate data using just two functions: melt and cast.'
   license_family: MIT
-
   license_file:
-    - {{ environ["PREFIX"] }}/lib/R/share/licenses/MIT
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
```
Comment (cbrueffer) [inline recipe/meta.yaml:44]: The quotes will be removed by the updater bot again, may as well leave them off.

File: recipe/meta.yaml (hunk 2 above)
Comment (cbrueffer) [reply recipe/meta.yaml:27]: Sounds reasonable.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -40,18 +38,29 @@ test:
...
+# Depends: R (>= 2.6.1)
+# Imports: plyr
+# License: MIT + file LICENSE
+# LazyData: true
+# NeedsCompilation: yes
+# Packaged: 2018-10-23 18:21:49 UTC; hadley
+# Author: Hadley Wickham [aut, cre]
+# Maintainer: Hadley Wickham <hadley@rstudio.com>
+# Repository: CRAN
+# Date/Publication: 2018-10-23 18:50:03 UTC
```
Comment (cbrueffer) [inline recipe/meta.yaml:66]: The metadata can also be left off IMHO, it's not very useful and gets stale very quickly.

Comment (jdblischak) [reply recipe/meta.yaml:66]: We decided to keep the metadata. Please see https://github.com/bgruening/conda_r_skeleton_helper/issues/25 and https://github.com/bgruening/conda_r_skeleton_helper/pull/31. If you would like to propose removing the metadata, please open an Issue in the helper script repo to discuss. I don't feel strongly about it. I agree it gets stale quickly, but on the other hand it is convenient for a quick spot check of the recipe. When it's not there, I usually end up having to manually look up the `DESCRIPTION` file on CRAN

File: recipe/meta.yaml (hunk 3 above)
Comment (jdblischak) [reply recipe/meta.yaml:44]: I am using the helper script with conda-build 3.21.7 to perform bulk updates. If we don't want the license file to be enclosed in strings, we need to either make the change to the cran skeleton in conda-build or the helper script.

**Comment by jdblischak** [comment]
@cbrueffer Thanks for the review. Much appreciated! As I put in my replies, a few of your comments relate to the behavior of the cran skeleton and the helper script. We need these tools to work for us. If they are unsatisfactory, let's change them to do what we need! I've had good success recently getting my changes to the cran skeleton merged to conda-build. I'd prefer to upstream our preferences so that we can eventually deprecate the helper script. I made some notes for myself on how to hack on [conda-build](https://github.com/jdblischak/jdb-conda/blob/main/docs/develop-conda-build.md), and I'm of course happy to assist.

**Comment by jdblischak** [comment]
Merging. Let's continue the discussion of the skeleton formatting at [bgruening/conda_r_skeleton_helper/issues](https://github.com/bgruening/conda_r_skeleton_helper/issues)

**Pull request merged**